cargo-apk/src/apk.rs

```diff
@@ -19,9 +20,23 @@ pub struct ApkBuilder<'a> {
     build_targets: Vec<Target>,
 }
 
+struct ConfigEnv(Option<LocalizedConfig>);
+impl Env for ConfigEnv {
+    fn var(&self, key: &str) -> std::result::Result<std::borrow::Cow<'_, str>, std::env::VarError> {
+        match &self.0 {
+            Some(config) => config
+                .resolve_env(key)
```

Comment:
Yeah it does, I'll improve the docs in https://github.com/dvc94ch/cargo-subcommand/pull/12. I'm not very fond of the name (perhaps just `.env()` too?), suggestions welcome!

Reply:
In fact it doesn't always. Only when:

- A key is set in both `[env]` and the global environment, and;
- `force = false`.

Reply:
Alternatively I'm thinking about making this function return `Option<(String, bool)>` where the boolean represents the `force` flag instead of falling back to `std::env`. Then it is up to the caller to decide how to deal with it, as we already have a "fallback" here when no `.cargo/config.toml` is used.

Reply:
Proper fallbacks are now implemented for every case. Documentation is updated in `cargo-subcommand` and tests are implemented there to make sure we adhere to it.